packages/core/src/index.ts

```diff
@@ -12,10 +12,10 @@ export { FilterCore, FilterCodecs } from "./lib/filter/index.js";
 export * as waku_light_push from "./lib/light_push/index.js";
 export { LightPushCodec, LightPushCore } from "./lib/light_push/index.js";
 
-export * as waku_store from "./lib/store/index.js";
-export { StoreCore } from "./lib/store/index.js";
-
-export { PageDirection } from "./lib/store/index.js";
+export * as waku_store from "./lib/store/v3/index.js";
+export { StoreCore } from "./lib/store/v3/index.js";
+export * as waku_store_v2 from "./lib/store/v2/index.js";
```

Comment:
I think we are good to remove `v2` as per discussion [here](https://discord.com/channels/1110799176264056863/1253380615231766539/1254720417944440882)